### PR TITLE
#134 Gas Station Problem Solution in cpp

### PR DESCRIPTION
**Pull request by psyduck1203**

### CODE REVIEW

File: #134/Cpp/Solution.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Solution {
```
Comment (aayush26) [inline #134/Cpp/Solution.cpp]: As per "Step 2: Folder structure and conventions" in contribution guidelines, please address #4.

**Review by aayush26 — CHANGES_REQUESTED** [review]
Minor comment: lets have folder as #134/cpp/Solution.cpp instead of #134/Cpp/Solution.cpp

File: #134/Cpp/Solution.cpp (hunk 1 above)
Comment (psyduck1203) [reply #134/Cpp/Solution.cpp]: Yeah Sure !!!

**Comment by psyduck1203** [comment]
Screenshot_2020-10-04 (6) Gas Station - LeetCode

File: #134/Cpp/Solution.cpp (hunk 1 above)
Comment (aayush26) [reply #134/Cpp/Solution.cpp]: LGTM

**Comment by aayush26** [comment]
Thanks for the contribution and addressing the comments. 
Have a great hacktoberfest ahead.

**Pull request merged**